src/Tooltip/examples/positions.js

```diff
@@ -0,0 +1,44 @@
+class Example extends React.Component {
```

Comment:
why not making this example more interactive by adding a dropdown to let the user change the current position without touching the code?